bin/rcm-release-data.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+
+here = sys.path[0]
+if here not in ('/usr/bin', '/bin', '/usr/local/bin'):
+    # git checkout
+    toplevel = os.path.dirname(here)
+    sys.path[0] = toplevel
+
+from distutils.sysconfig import get_python_lib
+
+sys.path.insert(1, os.path.join(get_python_lib(), "rcm_release"))
```

Comment:
The `rcm_release` library should just be a dependency in `setup.py` (i.e., this line and the import before it should be removed).